bundles/org.openhab.binding.mybmw/src/main/java/org/openhab/binding/mybmw/internal/handler/MyBMWProxy.java

```diff
@@ -322,12 +323,12 @@ public synchronized boolean updateToken() {
             /*
              * Step 2) Calculate values for base parameters
              */
-            String verfifierBytes = Converter.getRandomString(64);
+            String verfifierBytes = StringUtils.getRandomAlphabetic(64).toLowerCase();
```

Comment:
It seems this method includes a-z and A-Z, whereas the method being replaced included only a-z?

Reply:
`Converter.getRandomString()` is lowercase only (chars 97-122  is lowercase and 65-90  is uppercase)

`StringUtils.getRandomAlphabetic()` holds a set of both a-z and A-Z so it must be lowered to be an equal replacement.

Reply:
I must have been too sleepy, I completely missed your `toLowerCase()`.